docs/src/SUMMARY.md

```diff
@@ -3,6 +3,7 @@
 - [Getting started](./getting-started.md)
   - [Installation](./install-guide.md)
     - [Building from source](./build-from-source.md)
+    - [GitHub CI Action](./install-github-ci.md)
```

Comment:
Did you forget to add this file to this PR?

Reply:
Fixed with c5a50dc841578cb47019b4711712e36b4a551fce